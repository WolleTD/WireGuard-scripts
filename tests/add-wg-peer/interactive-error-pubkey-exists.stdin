peer2
aWosQEdndReq6diLeGq2BKKl0fFGA7v/3Qbwu9pbYxg=
n
