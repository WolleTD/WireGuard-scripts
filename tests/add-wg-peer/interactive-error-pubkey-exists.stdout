Add WireGuard peer\(s\) to infile
Name: PublicKey: A peer with this PublicKey already exists!
Add another peer\? \[y/N\] 
